docs/_config.yml

```diff
@@ -32,3 +32,6 @@ exclude:
   - Gemfile
   - Gemfile.lock
   - art
+
```

Comment:
Sorry but you cannot include plug-ins on github.  This will have to be removed,

Reply:
Good spot. I have created a task to remove this: https://jira.corp.adobe.com/browse/UMAPI-481.